### PR TITLE
Fix `setup_data()`

### PR DESCRIPTION
**Pull request by howardbaik**
I fixed a couple of bugs in `setup_data()` where I've added inline explanations of the fixes.

### CODE REVIEW

File: R/00-setup_data.R [hunk 1]
```diff
@@ -13,7 +13,7 @@
 #'   dplyr::select(c("Day00_RepA", "Day05_RepA", "Day22_RepA", "Day22_RepB", "Day22_RepC")) %>%
 #'   as.matrix()
 #'
-#' gimap_dataset <- setup_data(counts = example_counts_data)
+#' gimap_dataset <- setup_data(counts = example_counts)
```
Comment (howardbaek) [inline R/00-setup_data.R:16]: Wrong variable name.

File: R/00-setup_data.R [hunk 2]
```diff
@@ -23,7 +23,7 @@ setup_data <- function(counts = NULL,
                        pg_ids = NULL,
                        pg_metadata = NULL,
                        sample_metadata = NULL) {
-  new_data <- gimap_data <- list(
+  new_data <- list(
```
Comment (howardbaek) [inline R/00-setup_data.R:26]: Unnecessary assignment

File: R/00-setup_data.R (hunk 1 above)
Comment (kweav) [reply R/00-setup_data.R:16]: Great catch! Thank you!

**Review by kweav — APPROVED** [review]
Thanks for these catches and for the inline comments!

**Review by cansavvy — APPROVED** [review]
Great! Thanks @howardbaek

**Pull request merged**